README.md

```diff
@@ -31,10 +31,19 @@ Nx has support for a lot of platforms, but in this workshop we'll be using mainl
 
 Make sure you have the following installed:
 
-- [Node.js](https://nodejs.org/en/download/) version 12 and up
+- Node.js version 14 (recommended, but other versions might be okay as well)
   - `node --version`
-- [Yarn (optional)](https://classic.yarnpkg.com/en/docs/install/)
+- [Yarn](https://classic.yarnpkg.com/en/docs/install/)
   - `yarn --version`
+- A Github account
+  - http://github.com
+
+Optional (these won't be necessary to follow the workshop, but might unlock some bonus/advanced labs for you):
```

Comment:
Isn't this mandatory for deployment of API to Heroku?

Reply:
they are, but since they have a second track, I thought we'd make them optional? (so people don't stress out before the workshop if they couldn't get docker running 😄 )